lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+                                      color: Color(0xffBA82DF),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 30),
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffF99E5C),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 30),
+                                InkResponse(
+                                  child: new Container(
+                                    width: 20,
+                                    height: 20,
+                                    decoration: new BoxDecoration(
+                                      color: Color(0xffE53935),
+                                      shape: BoxShape.circle,
+                                    ),
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                          onTap: () =>
+                              print('Tile Name'), // Debug: change to edit
+                        ),
+                      ],
+                    ),
+                  ),
+                  Container(
+                    decoration: BoxDecoration(
+                      border: Border(
+                        bottom: new BorderSide(color: Colors.black12),
+                      ),
+                    ),
+                    child: Column(
+                      children: [
+                        ListTile(
+                          title: Text('Voice Recorder',
+                              style:
+                                  TextStyle(fontSize: 14, color: Colors.black)),
+                          subtitle: Text('On',
+                              style: TextStyle(
+                                  fontSize: 12, color: Color(0xff7C7C7C))),
+                          trailing: Wrap(
+                            children: [
+                              Icon(Icons.keyboard_arrow_right,
+                                  size: 35.0, color: Colors.black54),
+                            ],
+                          ),
+                          onTap: () =>
+                              print('Tile Name'), // Debug: change to edit
+                        ),
+                      ],
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            Row(
```

Comment:
I see here we have a `Row` with a `SizedBox` and `RaisedButton` - is there a way to simplify this a bit more?